### PR TITLE
Paddle es

### PR DESCRIPTION
**Pull request by wangzelong0663**
Add paddle es version.

### CODE REVIEW

**Review by zenghsh3 — DISMISSED** [review]
LGTM

File: README.cn.md [hunk 1]
```diff
@@ -95,6 +95,7 @@ pip install parl
 # 算法示例
 - [QuickStart](examples/QuickStart/)
 - [DQN](examples/DQN/)
+- [ES](examples/ES/)
```
Comment (TomorrowIsAnOtherDay) [inline README.cn.md:98]: Please add hyperlink at this page: 
https://github.com/PaddlePaddle/PARL/blob/develop/docs/zh_CN/Overview.md 
Put it under the tab`前沿算法`.

File: examples/ES/README.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+```
+
+Then we can start the distributed training by running:
+
+
+```bash
+python train.py
+```
+
+Training result will be saved in `train_log` with training curve.
```
Comment (TomorrowIsAnOtherDay) [inline examples/ES/README.md]: `the` training curve.

File: examples/ES/actor.py [hunk 3]
```diff
@@ -0,0 +1,125 @@
...
+
+        obs = self.env.reset()
+        while True:
+            if np.random.uniform() < self.config['filter_update_prob']:
+                obs = self.obs_filter(obs[None], update=True)
+            else:
+                obs = self.obs_filter(obs[None], update=False)
+
+            action = self.agent.predict(obs)
+            action = action.detach().numpy()
```
Comment (TomorrowIsAnOtherDay) [inline examples/ES/actor.py]: This line should be inside the function `agent.predict`.

File: examples/ES/actor.py [hunk 4]
```diff
@@ -0,0 +1,125 @@
...
+        return_filter = self.obs_filter.as_serializable()
+        if flush_after:
+            self.obs_filter.clear_buffer()
+        return return_filter
+
+    def set_filter(self, new_filter):
+        self.obs_filter.sync(new_filter)
+
+
+if __name__ == '__main__':
```
Comment (TomorrowIsAnOtherDay) [inline examples/ES/actor.py]: The following lines can be removed.

File: examples/ES/mujoco_model.py [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+
+
+class MujocoModel(parl.Model):
+    def __init__(self, obs_dim, act_dim):
+        super(MujocoModel, self).__init__()
+
+        hid1_size = 256
+        hid2_size = 256
+
+        # For Ant-v2, it is better to use the weight_init below, which is same to the torch version 1.8.1.
```
Comment (TomorrowIsAnOtherDay) [inline examples/ES/mujoco_model.py]: Please remove the comment. Users may worry that they cannot reproduce the result with this initialization method.

File: examples/ES/README.md [hunk 6]
```diff
@@ -0,0 +1,45 @@
...
+
+### Mujoco games introduction
+Please see [here](https://github.com/openai/mujoco-py) to know more about Mujoco games.
+
+### Benchmark result
+
+<img src=".result/result_ES_paddle0.png" alt="result">
+<img src=".result/result_ES_paddle1.png" alt="result">
+
+Please note that for Ant-V2, it is better to use weight initialization as what the torch uses [here](https://pytorch.org/docs/stable/generated/torch.nn.Linear.html#torch.nn.Linear), or if once the Ant flips over in the beginning with other weight initialization sometimes, the highest score of the Ant may be about just 960.
```
Comment (TomorrowIsAnOtherDay) [inline examples/ES/README.md]: Remove this line as we discussed.

File: examples/ES/README.md (hunk 2 above)
Comment (wangzelong0663) [reply examples/ES/README.md]: fix

File: examples/ES/actor.py (hunk 4 above)
Comment (wangzelong0663) [reply examples/ES/actor.py]: fix

File: README.cn.md (hunk 1 above)
Comment (wangzelong0663) [reply README.cn.md:98]: ok

File: examples/ES/actor.py (hunk 3 above)
Comment (wangzelong0663) [reply examples/ES/actor.py]: fix

File: examples/ES/mujoco_model.py (hunk 5 above)
Comment (wangzelong0663) [reply examples/ES/mujoco_model.py]: fix

File: examples/ES/README.md (hunk 6 above)
Comment (wangzelong0663) [reply examples/ES/README.md]: fix

File: examples/ES/mujoco_agent.py [hunk 7]
```diff
@@ -37,7 +37,7 @@ def predict(self, obs):
         obs = np.expand_dims(obs, axis=0)
         obs = paddle.to_tensor(obs, dtype='float32')
         predict_actions = self.alg.predict(obs)
-        return predict_actions
+        return predict_actions.numpy()
```
Comment (TomorrowIsAnOtherDay) [inline examples/ES/mujoco_agent.py]: Do we need detach() here?

Comment (wangzelong0663) [reply examples/ES/mujoco_agent.py]: As for paddle, it can runs well without adding the detach(), but add detach() seems more reasonably and same to the torch. Fixed it.

**Review by TomorrowIsAnOtherDay — APPROVED** [review]
LGTM.
